pbkhrv/rtl_433-hass-addons
# add device_topic_suffix arg

## Summary



First, thanks for a great add-on :)

rtl_433 supports custom topic name configurations and the HA MQTT helper script recently got support for it. This PR makes this topic name configurable also in the HA add-on and maps it through to rtl_433_mqtt_autodiscovery script invocation.

## Alternatives Considered



Since this feature has been added downstream and the add-on wraps down-stream I think this is the most sane way of achieving this. Alternatives would be to run rtl_433_mqtt_home_assistant manually on a different node.

## Testing Steps

1. Set up rtl_433 to use a different device topic path. My use case was to remove id from the path since they change with every battery change. Example line in rtl_433.conf:
```
output mqtt://IP,user=USERNAME,pass=PASS,devices=rtl_433/localhost/devices[/type][/model][/subtype]/C[channel:0],events=rtl_433/localhost/events
```
3. Now observe that the rtl433 HA MQTT add-on stops making working autoconfig topics because it assembles the wrong topic path 
4. With this PR, set device_topic_suffix to the same value (rtl_433/localhost/devices[/type][/model][/subtype]/C[channel:0]) and it should work again.
